src/_tutorials/language/futures.md

```diff
@@ -367,17 +382,31 @@ written in a couple different ways.
 
   <?code-excerpt "misc/lib/tutorial/daily_news.dart (main-future-api-pass-print)"?>
   {% prettify dart %}
-  printDailyNewsDigest() =>
+  Future<void> printDailyNewsDigest() =>
       gatherNewsReports().then(print);
   {% endprettify %}
 
+You need to provide an argument to `then()`'s callback,
+even if the Future is of type `Future<void>`:
+
+<?code-excerpt "misc/lib/tutorial/daily_news.dart (main-future-api-then-no-arg)"?>
+```dart
+Future<void> future = printDailyNewsDigest();
+
+return future.then((_) {
+  // Code that doesn't use the `_` parameter...
```

Comment:
You you instead want to mention the convention behind use of `_` (or link to a section that does in the prose before this code excerpt)?

Reply:
No :-/ ... I remember search for it a while back and having a hard time finding it.

Reply:
I just grepped the site-www source code and can't find it there. A web search for [[unused dart parameter underscore](https://www.google.com/search?q=unused+dart+parameter+underscore)] came up with dart-lang/sdk#23050 and https://stackoverflow.com/questions/33146619. So it's a longstanding convention but, as far as I can tell, an undocumented one.

@munificent did you consider documenting `_` as a convention for unused parameter names?

Reply:
Is there a way to maybe also state that using _ here would be an error?